lib/src/http/common/request.dart

```diff
@@ -149,8 +150,9 @@ abstract class CommonRequest extends Object
   /// Whether or not to send the request with credentials.
   bool _withCredentials = false;
 
-  /// [Client] instance from which this request was created. Used in [clone] to
-  /// correctly tie the clone to the same client.
+  /// HttpClient instance from which this request was created. Used in [clone]
```

Comment:
The comment claims this is an `HttpClient`, but the code remains a `Client` which isn't in the same class hierarchy.